package.json

```diff
@@ -29,6 +29,7 @@
     "@aragon/wrapper": "^5.0.0-rc.20",
     "@sentry/browser": "^5.5.0",
     "@ungap/event-target": "^0.1.0",
+    "clipboard-polyfill": "^2.8.6",
```

Comment:
@bpierre should we use this dependency in aragonUI's `TextCopy` too? We're just using `document.execCommand()` right now there.

Reply:
Yes I think we might want to use it. [It’s not too big](https://bundlephobia.com/result?p=clipboard-polyfill) (at least compared to other dependencies we have!), and it gets around [implementation bugs](https://github.com/lgarron/clipboard-polyfill/blob/master/experiment/Conclusions.md) that are still present in current browsers.